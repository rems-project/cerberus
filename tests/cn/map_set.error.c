/*@

function (map<integer, bool>) write_to_012(integer foo, map<integer, bool> my_map) {
    my_map[0: true, 1: (false || false), 2: 3]
}

@*/

