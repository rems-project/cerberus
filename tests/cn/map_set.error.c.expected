return code: 2
tests/cn/map_set.error.c:3:24: error: the type name `bool' is not declared
function (map<integer, bool>) write_to_012(integer foo, map<integer, bool> my_map) {
                       ^
